testsuite/tests/clean/cache-files/test.py

```diff
@@ -0,0 +1,39 @@
+"""
+Verify that `alr clean --cache` works properly
+"""
```

Comment:
This test seems unrelated to this PR

Reply:
As discussed above, `shutil.rmtree` sometimes has trouble clearing the cache on Windows. This was the simplest way to verify those issues are not shared by `Alire.Directories.Force_Delete`.

I can split it into a separate PR (or drop it altogether) if you'd prefer.